src/components/Settings/index.tsx

```diff
@@ -136,22 +146,18 @@ export default function SettingsTab({
   )
 
   return (
-    <>
-      <Menu ref={node}>
-        <MenuButton
-          disabled={!isChainSupported || chainId !== connectedChainId}
-          isActive={isOpen}
-          onClick={toggleMenu}
-        />
-        {isOpenDesktop && !isMobile && <MenuFlyout>{Settings}</MenuFlyout>}
-      </Menu>
-      {isMobile && (
-        <Portal>
+    <Menu ref={node}>
+      <MenuButton disabled={!isChainSupported || chainId !== connectedChainId} isActive={isOpen} onClick={toggleMenu} />
+      {isOpenDesktop && <MenuFlyout>{Settings}</MenuFlyout>}
+      <Portal>
```

Comment:
is there a reason you moved portal outside of the mobile check?

Reply:
The createPortal api (which Portal uses under the hood) specifies child/container elements in a way that feels static to me. This isn't important to me though. 

https://react.dev/reference/react-dom/createPortal

Reply:
reverted this